### DISCUSSION TITLE
Solarleistung in 1.1.8 not available anymore?

### DISCUSSION THREAD

**Discussion by ThomasWollstadt** [LarsK1/hass_solvis_control#74] [General]
I did upgrade from 1.1.3 to 1.1.8 and it looks like some sensors (solar flow, solar heat exchanger temp in/out) to calculate solar power are not available anymore? Did I do something wrong during installation? Those were supported in 1.1.3... The release notes of 1.1.3 did indidcate solar power was a calculated value by the integration itself. Since I didn't find it I calculated it up to now myself by using the above sensors...

**Comment by LarsK1** [comment]
Hi @ThomasWollstadt ,
please try to configure the integration and make sure solar power is activated in the second options page. The entity shouldn't have changed...

**Comment by ThomasWollstadt** [comment]
Thanks for the good hint.. I didn't recognize the configuration option. 

 

Von: Lars Kusch ***@***.***> 
Gesendet: Freitag, 27. Dezember 2024 22:00
An: LarsK1/hass_solvis_control ***@***.***>
Cc: ThomasWollstadt ***@***.***>; Mention
***@***.***>
Betreff: Re: [LarsK1/hass_solvis_control] Solarleistung in 1.1.8 not
available anymore? (Discussion #74)

 

Hi @ThomasWollstadt <https://github.com/ThomasWollstadt>  ,
please try to configure the integration and make sure solar power is
activated in the second options page. The entity shouldn't have changed...

-
Reply to this email directly, view it on GitHub
<https://github.com/LarsK1/hass_solvis_control/discussions/74#discussioncomm
ent-11679998> , or unsubscribe
<https://github.com/notifications/unsubscribe-auth/AZFRA2DMW3SSJ4VQTIS6MQL2H
W5VLAVCNFSM6AAAAABUJBEI6WVHI2DSMVQWIX3LMV43URDJONRXK43TNFXW4Q3PNVWWK3TUHMYTC
NRXHE4TSOA> .
You are receiving this because you were mentioned.
<https://github.com/notifications/beacon/AZFRA2ACWIQ2JCM74C7J4RD2HW5VLA5CNFS
M6AAAAABUJBEI6WWGG33NNVSW45C7OR4XAZNRIRUXGY3VONZWS33OINXW23LFNZ2KUY3PNVWWK3T
UL5UWJTQAWI4P4.gif> Message ID:
***@***.***
***@***.***
hub.com> >

**Comment by ThomasWollstadt** [comment]
Hi, nachdem ich endlich mal wieder Leistung von der Solaranlage bekommen
habe stelle ich fest, dass der Sensor Leistung Solarthermie konstant -0,1kW
anzeigt. Muss ich hier noch etwas konfigurieren? In der Solvis App sehe ich
z.B 1.7kW.

 

Von: Lars Kusch ***@***.***> 
Gesendet: Freitag, 27. Dezember 2024 22:00
An: LarsK1/hass_solvis_control ***@***.***>
Cc: ThomasWollstadt ***@***.***>; Mention
***@***.***>
Betreff: Re: [LarsK1/hass_solvis_control] Solarleistung in 1.1.8 not
available anymore? (Discussion #74)

 

Hi @ThomasWollstadt <https://github.com/ThomasWollstadt>  ,
please try to configure the integration and make sure solar power is
activated in the second options page. The entity shouldn't have changed...

-
Reply to this email directly, view it on GitHub
<https://github.com/LarsK1/hass_solvis_control/discussions/74#discussioncomm
ent-11679998> , or unsubscribe
<https://github.com/notifications/unsubscribe-auth/AZFRA2DMW3SSJ4VQTIS6MQL2H
W5VLAVCNFSM6AAAAABUJBEI6WVHI2DSMVQWIX3LMV43URDJONRXK43TNFXW4Q3PNVWWK3TUHMYTC
NRXHE4TSOA> .
You are receiving this because you were mentioned.
<https://github.com/notifications/beacon/AZFRA2ACWIQ2JCM74C7J4RD2HW5VLA5CNFS
M6AAAAABUJBEI6WWGG33NNVSW45C7OR4XAZNRIRUXGY3VONZWS33OINXW23LFNZ2KUY3PNVWWK3T
UL5UWJTQAWI4P4.gif> Message ID:
***@***.***
***@***.***
hub.com> >

**Comment by LarsK1** [comment]
Hi @ThomasWollstadt,
ja, das ist leider richtig. Stammt aber aus einem Fehler von Solvis seit der letzten Firmware. In frühren Firmwareversionen ging das, jetzt leider nicht (mehr). Laut Solvis soll das aber (wieder) behoben werden...
